include/Contact.php

```diff
@@ -662,4 +664,34 @@ function posts_from_contact($a, $contact_id) {
 
 	return $o;
 }
+
+/**
+ * @brief Returns a formatted location string from the given profile array
+ *
+ * @param array $profile Profile array (Generated from the "profile" table)
+ *
+ * @return string Location string
+ */
+function formatted_location($profile) {
```

Comment:
I'm not sure if we should have this function in include/Contacts.php   There are already some some format_* functions in include/text.php. Even it is a very large file it should be placed there for consistency reasons. What are you thinking?